### PR TITLE
Trust Gateway Section

### PR DESCRIPTION
**Pull request by radamson**
Opening a PR on behalf of Greg Shemancik for the Gateway section.

### CODE REVIEW

File: docs/gateway.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+# Trust Gateway
+
+## Purpose
+
+The Digital Documentation COVID-19 Certificate Gateway (DDCCG) is an extended
+version of the [European Union Digital COVID Certificate Gateway (EUDCCG)](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_v2_en.pdf),
+which has enhanced functionality allowing for interoperability between
+multiple gateways and supports of the DDCC specification.
```
Comment (mickohanlon23) [inline docs/gateway.md]: From MJ:
```suggestion
multiple gateways in support of the DDCC specification.
```

File: docs/gateway.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+documented network of trust gateways, supporting all major COVID-19 vaccine credential certificates. 
+
+## Documentation and Specification
+
+For more on the intent of the DDCCG, its design and architecture, and use cases
+supported, [please visit the following linked documentation](https://github.com/WorldHealthOrganization/ddcc-gateway/blob/main/docs/Architecture.md).
+The full [DDCCG public repository can be found linked here](https://github.com/WorldHealthOrganization/ddcc-gateway)
+for reference.
+
+Resources:
```
Comment (gjshemancik) [inline docs/gateway.md]: Remove Resources section when published to Wiki - is simply for reference, all resources have been hyperlinked

File: docs/gateway.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+-	Access to metadata content (e.g. value sets/codings, business rules) with explicit adherence to the HL7 FHIR specification
+-	Explicit means for revocation of digital COVID certificates
+-	Optional support of online verification and validation workflows
+
+The role of the DDCCG is to enable the creation of trust gateways by additional
+organizations beyond the EU Member States participating in the EUDCCG,
+expanding the network of operators and trusted parties to form a federated and
+documented network of trust gateways, supporting all major COVID-19 vaccine credential certificates. 
+
+## Documentation and Specification
```
Comment (gjshemancik) [inline docs/gateway.md:22]:
```suggestion
## COVID-19 Specific Documentation and Specification
```

File: docs/gateway.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 5
+---
+
+# Trust Gateway
+
+## Purpose
+
```
Comment (gjshemancik) [inline docs/gateway.md:8]: //Add text to make intro generic:

The Trust Gateway is designed as an open-source implementation of a trust network infrastructure for health credentialing.

Comment (gjshemancik) [inline docs/gateway.md:8]: //Add New Section titled:

##COVID-19 Specific Reference Implementations

File: docs/gateway.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+The Digital Documentation COVID-19 Certificate Gateway (DDCCG) is an extended
+version of the [European Union Digital COVID Certificate Gateway (EUDCCG)](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_v2_en.pdf),
+which has enhanced functionality allowing for interoperability between
+multiple gateways in support of the DDCC specification.
+
+While the DDCCG is designed to support the DDCC specification, 
+it also supports various existing digital COVID certificates
+including EUDCC, Smart Health Cards, DIVOC, and ICAO.
```
Comment (gjshemancik) [inline docs/gateway.md]: The Trust Gateway specification is based upon the [Digital Documentation COVID-19 Certificate Gateway (DDCCG)](https://github.com/WorldHealthOrganization/ddcc-gateway/blob/main/docs/Architecture.md) and [European Union Digital COVID Certificate Gateway](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_v2_en.pdf) (EUDCCG) specification, which were developed as the trust network infrastructure for COVID-19 specific vaccine credentialing. DDCCG is an extended version of EUDCCG, which has enhanced functionality allowing for interoperability between multiple gateways and supports of the DDCC specification. While the DDCCG is designed to support the DDCC specification, it also supports various existing digital COVID certificates including EUDCC, Smart Health Cards, DIVOC, and ICAO.

Comment (gjshemancik) [reply docs/gateway.md]: Condensed and updated language for lines 9-16 above^

Attempted to make generic entry to Trust Networks, and identify DDCC and EU specifications as covid examples to reference and base off of

**Review by nkarip — APPROVED** [review]

**Pull request merged**